code/src/java/plugin/lsttokens/deprecated/ChooseFeatSelectionToken.java

```diff
@@ -129,39 +129,8 @@ private String getFullName()
 	@Override
 	public String[] unparse(LoadContext context, CDOMObject cdo)
 	{
-		ChooseInformation<?> tc =
-				context.getObjectContext()
-					.getObject(cdo, ObjectKey.CHOOSE_INFO);
-		if (tc == null)
-		{
-			return null;
-		}
-		if (!tc.getName().equals(getTokenName()))
-		{
-			// Don't unparse anything that isn't owned by this SecondaryToken
-			/*
-			 * TODO Either this really needs to be a check against the subtoken
-			 * (which thus needs to be stored in the ChooseInfo) or there needs
-			 * to be a loadtime check that no more than once CHOOSE subtoken
-			 * uses the same AssociationListKey... :P
-			 */
-			return null;
-		}
-		if (!tc.getGroupingState().isValid())
-		{
-			context.addWriteMessage("Invalid combination of objects"
-				+ " was used in: " + getParentToken() + Constants.COLON + getTokenName());
-			return null;
-		}
-		StringBuilder sb = new StringBuilder();
-		sb.append(tc.getLSTformat());
-		String title = tc.getTitle();
-		if (!title.equals(getDefaultTitle()))
-		{
-			sb.append("|TITLE=");
-			sb.append(title);
-		}
-		return new String[]{sb.toString()};
+		//Nothing here anymore - defer to ABILITYSELECTION

```

Comment:
Okay, let me take that as a TODO to get more tests in place to ensure it is working properly.  This can be held until that integration is complete or this PR closed and I can open another later... either way works for me